playground/AzureFunctionsEndToEnd/AzureFunctionsEndToEnd.Functions/Program.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.Azure.Functions.Worker.OpenTelemetry;
+using Microsoft.Extensions.Hosting;
+using OpenTelemetry;
+using Microsoft.Extensions.DependencyInjection;
+
+var host = new HostBuilder()
+    .AddServiceDefaults()
+    .ConfigureFunctionsWorkerDefaults()
+    .ConfigureServices(services =>
+    {
+        services.AddOpenTelemetry()
+            .UseFunctionsWorkerDefaults()
+            .UseOtlpExporter();
```

Comment:
How come this is here and not in `.AddServiceDefaults`?

Reply:
IHostBuilder vs IHostApplicationBuilder

Reply:
AddServiceDefaults (called above) is also implemented in terms of IHostBuilder. It is in the `HostBuilderExtensions.cs` file.